pyproject.toml

```diff
@@ -138,9 +138,7 @@ omit = [
     "kedro/framework/cli/hooks/specs.py",
     "kedro/framework/hooks/specs.py",
     "tests/*",
-    "kedro/io/core.py",  # TODO: temp fix for removing datasets, need to resolve this before 0.19
-    "kedro/runner/parallel_runner.py",
```

Comment:
Should `"kedro/runner/parallel_runner.py"` be removed from here as well? I thought only the line above was a temporary fix.

Reply:
No, it shouldn't. It's causing the code coverage on windows to fail 😓